README.md

```diff
@@ -30,6 +33,15 @@ The SageMaker HyperPod CLI is a tool that helps submit training jobs to the Amaz
   - Or you can follow the [Readme under helm_chart folder](https://github.com/aws/sagemaker-hyperpod-cli/blob/main/helm_chart/readme.md) to install Kubeflow Training Operator.
 - Configure [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) to point to the correct region where your HyperPod clusters are located. 
 
+## Platform Support
+
+SageMaker HyperPod CLI currently supports Linux and MacOS platforms. Windows platform support is working in progress
+
+## ML Framework Support
+
+SageMaker HyperPod CLI currently supports start training job with:
+- PyTorch ML Framework. Version requirements: PyTorch > 1.10
```

Comment:
should it be > 1.0.0 ?

Reply:
It should be > 1.10 - https://discuss.pytorch.org/t/how-to-install-torchrun/135138